scripts/ci/linux/setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+echo "Configuring build type '$BUILD_TYPE'"
+mkdir build
+
+conda update -n base -c defaults conda
+conda install cmake ninja conda-build compilers -y
+
+if [ "$BUILD_TYPE" == "fixed" ]; then
```

Comment:
Only problematic on Linux? Or just overkill to matrix this to other OSs also?

Reply:
I was replicating what we had on AzP. It would be a good idea to spread it to the other builds too.

Reply:
The idea of 'fixed' is a fixed GDAL dependency that doesn't change, which should pin most of the other PDAL dependencies. This needs to specified individually for each setup.sh of all builders.

Reply:
Added floating and fixed to osx and windows